site/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es6",
```

Comment:
Please don't change this! This means we generate es6 and this would break everyone who uses a browser that doesn't support es6.